### PR TITLE
OSDOCS10255: Updated the release notes to add cert manager-1.13.1 section

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13 - 4.16

Issue:  [OSDOCS-10255](https://issues.redhat.com/browse/OSDOCS-10255)

Link to docs preview: [Docs Preview](https://75962--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-release-notes.html#cert-manager-operator-release-notes-1-13-1)

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 1]
```diff
@@ -24,17 +42,20 @@ The following advisory is available for the {cert-manager-operator} 1.13.0:
...
 === New features and enhancements
 
 * You can now manage certificates for API Server and Ingress Controller by using the {cert-manager-operator}.
-For more information, see xref:../../security/cert_manager_operator/cert-manager-creating-certificate.adoc#cert-manager-creating-certificate[Configuring certificates with an issuer].
+For more information, see xref:../../security/cert_manager_operator/cert-manager-creating-certificate#cert-manager-creating-certificate[Configuring certificates with an issuer]. 
 
-* With this release, the scope of the {cert-manager-operator}, which was previously limited to the {product-title} on AMD64 architecture, has now been expanded to include support for managing certificates on {product-title} running on {ibm-z-name} (`s390x`), {ibm-power-name} (`ppc64le`) and ARM64 architectures.
+* With this release, the scope of the cert-manager Operator for Red Hat OpenShift, which was previously limited to the OpenShift Container Platform on AMD64 architecture, has now been expanded to include support for managing certificates on OpenShift Container Platform running on IBM Z® (s390x), IBM Power® (ppc64le) and ARM64 architectures.
```
Comment (snarayan-redhat) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:51]: Why is this content changed?

Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:51]: While creating the PR with the previous commit, it was deleted, I had to manually add it.

Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc]: I have reverted the changes

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 2]
```diff
@@ -24,6 +42,7 @@ The following advisory is available for the {cert-manager-operator} 1.13.0:
 Version `1.13.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.3`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.13/#v1133[cert-manager project release notes for v1.13.0].
 
 [id="cert-manager-operator-new-features-1-13"]
+[id="cert-manager-operator-new-features-1-13-0"]
```
Comment (snarayan-redhat) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:45]: Retain just 1 id field

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 3]
```diff
@@ -34,7 +53,9 @@ For more information, see xref:../../security/cert_manager_operator/cert-manager
 * With this release, you can use DNS over HTTPS (DoH) for performing the self-checks during the ACME DNS-01 challenge verification. The DNS self-check method can be controlled by using the command line flags, `--dns01-recursive-nameservers-only` and `--dns01-recursive-nameservers`.
 For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.html#cert-manager-override-arguments_cert-manager-customizing-api-fields[Customizing cert-manager by overriding arguments from the cert-manager Operator API].
 
+
 [id="cert-manager-operator-1-13-CVEs"]
+[id="cert-manager-operator-1-13-0-CVEs"]
```
Comment (snarayan-redhat) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:58]: Retain just 1 id field

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 4]
```diff
@@ -12,7 +12,25 @@ These release notes track the development of {cert-manager-operator}.
...
+* link:https://access.redhat.com/errata/RHEA-2024:2849[RHEA-2024:2849]
+
+Version `1.13.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.6`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.13#v1136[cert-manager project release notes for v1.13.6].
+
+[id="cert-manager-operator-1-13-1-CVEs"]
+=== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2023-45288[CVE-2023-45288]
+* link:https://access.redhat.com/security/cve/CVE-2023-48795[CVE-2023-48795]
+* link:https://access.redhat.com/security/cve/CVE-2024-24783[CVE-2024-24783]
```
Comment (snarayan-redhat) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:31]: there are 9 CVEs in the advisory. Also, I do not see any common ones either. Is this intentional?

**Review by snarayan-redhat — COMMENTED** [review]
Left few comments that need to be fixed.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 3 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:58]: Ack

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 4 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:31]: I have added the ones that were given by SME.

Comment (snarayan-redhat) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:31]: Can you please verify with SME?

Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:31]: Seems like some issue with errata, but the ones provided from the swarup are the ones fixed. And swarup mentioned as long as we keep 45288, for the other CVEs, we can decide which ones we need to add.

Comment (snarayan-redhat) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:31]: thanks

**Comment by snarayan-redhat** [comment]
for 4.13 and 4.14, please create manual CPs

**Pull request merged**